cl/search/management/commands/cl_dumpdata.py

```diff
@@ -34,42 +44,85 @@ def handle(self, *args, **options):
         fmt = options['format']
 
         self.stdout.write(
-            'Generating dump of up to %s randomly selected Opinions' % n
+            'Generating dump of up to %s randomly selected Opinions...' % n
         )
+
+        pks, cluster_pks = self._select_pks(n)
+        if n > len(pks):
+            self.stdout.write(
+                ' n > number of Opinions, serializing all (n=%s)' % len(pks)
+            )
+
+        try:
+            self._serialize(Opinion, fmt, Q(id__in=pks))
+
+            self._serialize(
+                OpinionsCited,
+                fmt,
+                Q(citing_opinion__in=pks) | Q(cited_opinion__in=pks)
```

Comment:
Interesting problem. Wouldn't have occurred to me that this would be the needed solution. Nice.